provider/openstack/provider.go

```diff
@@ -718,7 +718,7 @@ func (e *Environ) getKeystoneDataSource(mu *sync.Mutex, datasource *simplestream
 		}
 	}
 
-	url, err := makeServiceURL(e.client, keystoneName, nil)
+	url, err := makeServiceURL(e.client, keystoneName, "v2.0", nil)
```

Comment:
Dumb question - we ask for "2.0". Should we fall back to "1.0" if 2.0 not available? What happens if they try on an openstack version that doesn't have the newer 2.0 API?

Also, why pass in "v2.0" and not "2.0"? What does using the "v" prefix gain in return for the added complexity?